source/aggregation/atlas-search.txt

```diff
@@ -66,12 +65,12 @@ Search queries by using the Aggregation Builder:
 To create a ``$search`` stage in your aggregation pipeline, perform the
 following actions:
 
-1. Create an array to store the pipeline stages
+1. Create an array to store the pipeline stages.
 
-#. Call the ``Stage::search()`` method to create the Atlas Search stage
+#. Call the ``Stage::search()`` method to create the Atlas Search stage.
 
 #. Within the body of the ``search()`` method, use methods from the
-   ``Search`` builder class to construct your Search query criteria
+   ``Search`` builder class to construct your Search query criteria.
```

Comment:
I'll note that some of the bullets in `source/aggregation.txt` omitted trailing periods. Should these all be consistent?

Reply:
Thanks for catching that - its a style guide convention that I forgot to abide by in that PR. so fixing it here.
Lists of complete sentences should use periods.